formulas/dnsmasq/files/protect-dns-resolver-config

```diff
@@ -0,0 +1,19 @@
+eval "$(
+    funcBody=$(declare -f make_resolv_conf)
+    echo "${funcBody/make_resolv_conf/make_resolv_conf_real}"
+);"
+
+make_resolv_conf() {
+    resolv_conf=$(readlink -f "/etc/resolv.conf" 2>/dev/null) || resolv_conf="/etc/resolv.conf"
+    cp -p $resolv_conf /etc/resolv.conf.tmp || return 1
+    make_resolv_conf_real
+    echo "# Generated at $(date)" >> resolv_conf
```

Comment:
Missing `$` in front of `resolv_conf`, but also should be quoted and changed to camel case.

Missing `$` in front of the variable elsewhere in this function.